src/twisted/internet/test/test_process.py

```diff
@@ -582,7 +582,7 @@ def processExited(self, reason):
         # Only required if the test fails on systems that support
         # the process module.
         if process is not None:
-            for pid, handler in items(process.reapProcessHandlers):
+            for pid, handler in list(process.reapProcessHandlers.items()):
```

Comment:
Why `list()`?

Reply:
`unregisterReapProcessHandler()` executes `del reapProcessHandlers[pid]`, which modifies the dictionary we're iterating over and `dict` does not support that.

Reply:
Since it's a test… meh

Reply:
I'll leave in the `list()` then: in a test it is more important to be obviously correct than to be efficient.